.github/helpers/contracts/dummy-dao/proposal.js

```diff
@@ -87,10 +87,10 @@ async function proposal() {
 
   // initiate Voting contract
   let VotingContract;
-  if (isERC20Voting === 'erc20') {
+  if (isTokenVoting === 'token') {
```

Comment:
not a fan of this. `isTokenVoting` should be boolean or it should be named: `votingType` to make sense of it

Reply:
Fixed although outside this PR's scope